content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
+* Added `when` field support for config groups.
+{{</features>}}
+
+{{<changes>}}
+* Kotsadm - New Support bundles will be shown at the top of the list instead of the bottom.
+* Kotsadm - Showing warnings on the Version History page in the Admin Console when installing an application with invalid YAML.
+* Kotsadm - Added a page title for the 'View files' and 'Config' tabs.
+* Removed the KOTS metadata that was added to all application manifests. Which included the local application and the version sequence injected as annotations to all deployed manifests.
+* Show the airgap upload progress percentage more granularly.
+* Removed the "Re-analyze" button from the troubleshoot support bundle analysis page.
+* Auto deploy the app's first version when installing the app and no preflight checks are available or preflight checks pass.
```

Comment:
I thought we already did this? Or is this just for existing clusters? What was the experience like when we didn't do this and no preflights were configured?

Reply:
Hmmm, you're right, we should probably move this to the "Fixes" section.
This was not working for:
1- Automated installs using the CLI.
2- For apps with no preflight checks.
3- If "Continue" is not clicked in the preflights page.

Reply:
Sounds like it should be in the "Fixes" section. 

Still don't quite understand what happened. But if you pair that up with the 3 items above, that sounds good to me.